app/models/mixins/authentication_mixin.rb

```diff
@@ -273,6 +273,27 @@ def authentication_check(*args)
     return status == :valid, details
   end
 
+  def authentication_check_queue(*args)
```

Comment:
@jrafanie `authentication_check_types_queue` works just as great! Thanks for pointing me to this method.
However, the only thing that needs to happen before we can use this method is to make sure that ems types that have authentication types other than `default` should have this method available - `authentications_to_validate`.